frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/header/cover_title.dart

```diff
@@ -83,7 +83,7 @@ class _InnerCoverTitleState extends State<_InnerCoverTitle> {
         editorState.selection = null;
       }
     });
-    _requestFocusIfNeeded(widget.view, null);
+    // _requestFocusIfNeeded(widget.view, null);
```

Comment:
I've been trying to make this work, but I haven't found a solution yet. If anyone knows how I can fix it, please help!

https://github.com/google/flutter.widgets/issues/337